llama_stack/distribution/datatypes.py

```diff
@@ -234,6 +234,28 @@ class AuthenticationConfig(BaseModel):
     )
 
 
+class QuotaPeriod(str, Enum):
+    DAY = "day"
+
+
+class QuotaType(str, Enum):
+    SQLITE = "sqlite"
+
+
+class QuotaSqliteConfig(BaseModel):
```

Comment:
Please look into llama_stack/providers/utils/kvstore/config.py to re-use SqliteKVStoreConfig or build a new class to inherits from SqliteKVStoreConfig

Reply:
I reused `SqliteKVStoreConfig`, thanks!